utils/indigo-depict/main.c

```diff
@@ -955,7 +955,7 @@ int main(int argc, char* argv[])
         _prepare(obj, p.aromatization);
         if (p.action == ACTION_LAYOUT)
         {
-            indigoLayout(obj);
+            //indigoLayout(obj);

```

Comment:
This file probably should not be changed.

Reply:
fixed